File: JavaBasic/115_sum_digits.java
// 115	115_sum_digits.java	Find the sum of all digits of a given number.
public class _115_sum_digits {
    public static void main(String[] args) {
        int num = 4567, sum = 0, temp = num;
        while (temp > 0) { sum += temp%10; temp /= 10; }
        System.out.println("Number: " + num + " | Sum of digits: " + sum);
    }
}

File: JSBASIC/115_sum_digits.js
// 115	115_sum_digits.js	Find the sum of all digits of a given number.

let num = 4567;
let sum = 0;
let temp = num;
while (temp > 0) {
    sum += temp % 10;
    temp = Math.floor(temp / 10);
}
console.log("Number:", num);
console.log("Sum of digits:", sum);
